feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file also added in the feature1 branch
+
```

Comment:
It's really good that you have decided to add this code.

Reply:
Thanks for the comment Kobe Koki 24